nipy/algorithms/registration/tests/test_fmri_realign4d.py

```diff
@@ -23,6 +25,9 @@
 from ..affine import Rigid
 
 im = load_image(funcfile)
```

Comment:
Sorry - this isn't fixing what you did, but what was there before.  Would you mind capitalizing these to `IMG` and `IMAGES` to make it clear they are module globals.